### ISSUE TITLE
50+ Slides can't work properly 

### ISSUE THREAD

**Issue by zahoorcs12** [nateplusplus/pushIn-js#1]
hi i am using svg slides in that plugin and i have 50+ plus slides after scroll on 20 slides other slides responding time is too much slow

**Comment by nateplusplus** [comment]
Oh my goodness, I am so sorry for never responding, @zahoorcs12! To be honest, I didn't realize this plugin was being used by other people. I built this when I was a beginner coder learning the ropes so I have no doubt the performance was horrible with over 50 slides!! I will look back at this code to see if I can improve it.

Did you find a solution for your project?

**Comment by nateplusplus** [comment]
@zahoorcs12 thanks again for flagging this.

After noticing the attention that this plugin is getting here on GitHub, I have made a lot of performance improvements over the past month.

Looking at the latest release, Chrome lighthouse is showing resources on the demo page loading in less than .3s, the source files are less than 5kb. I ran a test where I could scroll through over 100 layers without a problem.

I think this issue has been resolved so I am going to close this.
